pr_watch/tasks.py

```diff
@@ -46,10 +50,11 @@ def watch_pr():
     organization on the watched repository
     """
     team_username_list = get_username_list_from_team(settings.WATCH_ORGANIZATION)
```

Comment:
Theoretically, we could get rate-limited for this request as well, so you could move it inside the try/except block.

I know the limit for this kind of query is much higher than for search queries, but we now mostly have non-search queries.